### ISSUE TITLE
scandir.zip doesn't install on a Docker image of Ubuntu 16.04

### ISSUE THREAD

**Issue by AvdN** [benhoyt/scandir#72]
with `getpip.py` from https://pip.pypa.io/en/stable/installing/ in a subdirectory `pip` and this `Dockerfile`:
```
FROM ubuntu:16.04

MAINTAINER Anthon van der Neut <a.van.der.neut@ruamel.eu>

ENV DEBIAN_FRONTEND noninteractive

RUN apt-get update && apt-get install -y \
  python3 \
&& apt-get clean \
&& rm -rf /var/lib/apt/lists/*

# this gets you the latest pip
COPY pip/get-pip.py /tmp/get-pip.py
RUN python3 /tmp/get-pip.py

RUN pip3 install virtualenv

RUN virtualenv /tmp/scandirtest
RUN /tmp/scandirtest/bin/pip install 'pathlib2>=2.1.0'
```

running `docker build .` will result in an error while trying to install `scandir` from its ZIP file.
```
....
Step 9 : RUN /tmp/scandirtest/bin/pip install 'pathlib2>=2.1.0'
 ---> Running in 3d3cf0e88ee7
Collecting pathlib2>=2.1.0
  Downloading pathlib2-2.2.0-py2.py3-none-any.whl
Collecting six (from pathlib2>=2.1.0)
  Downloading six-1.10.0-py2.py3-none-any.whl
Collecting scandir (from pathlib2>=2.1.0)
  Downloading scandir-1.4.zip
Exception:
Traceback (most recent call last):
  File "/tmp/scandirtest/lib/python3.5/site-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "/tmp/scandirtest/lib/python3.5/site-packages/pip/commands/install.py", line 335, in run
    wb.build(autobuilding=True)
  File "/tmp/scandirtest/lib/python3.5/site-packages/pip/wheel.py", line 749, in build
    self.requirement_set.prepare_files(self.finder)
  File "/tmp/scandirtest/lib/python3.5/site-packages/pip/req/req_set.py", line 380, in prepare_files
    ignore_dependencies=self.ignore_dependencies))
  File "/tmp/scandirtest/lib/python3.5/site-packages/pip/req/req_set.py", line 620, in _prepare_file
    session=self.session, hashes=hashes)
  File "/tmp/scandirtest/lib/python3.5/site-packages/pip/download.py", line 821, in unpack_url
    hashes=hashes
  File "/tmp/scandirtest/lib/python3.5/site-packages/pip/download.py", line 663, in unpack_http_url
    unpack_file(from_path, location, content_type, link)
  File "/tmp/scandirtest/lib/python3.5/site-packages/pip/utils/__init__.py", line 599, in unpack_file
    flatten=not filename.endswith('.whl')
  File "/tmp/scandirtest/lib/python3.5/site-packages/pip/utils/__init__.py", line 499, in unzip_file
    fp = open(fn, 'wb')
UnicodeEncodeError: 'ascii' codec can't encode characters in position 58-59: ordinal not in range(128)

```
This is caused by files only needed for testing being included in the zip file. 
For my projects I leave out test data, if someone wants that they can check the data (and the test files) out from the repository.

**Comment by benhoyt** [comment]
Ah, interesting -- yes, you're right, thanks! The `test/testdir` directory shouldn't be included in the zip file. I'll tighten up the `MANIFEST.in` rules and upload a new package.

**Comment by benhoyt** [comment]
Thanks @AvdN for the bug report and @andrask for the pull request. I've fixed and uploaded a new version (1.5) to PyPI. Let me know if this works for you.
